roachtest: Update psycopg block list

spilchen opened the pull request:
This allows failures for some tests in the pscyopg tests that were due to the recent autocommit_before_ddl change in #141145. Rationale for each failing test are included as comments in the block list.

I will backport this in: https://github.com/cockroachdb/cockroach/pull/141987 https://github.com/cockroachdb/cockroach/pull/141851.

Epic: none
Release note: none
Closes #141884

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/142047)
<!-- Reviewable:end -->

spilchen committed:
roachtest: Update psycopg block list

This allows failures for some tests in the pscyopg tests that were due
to the recent autocommit_before_ddl change in #141145. Rationale for
each failing test are included as comments in the block list.

Epic: none
Release note: none
Closes #141884

Dedej-Bergin reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/142047#-:-OK3dpYB-km6r-qnP-mp:bnfp4nl)* status: <span>:shipit:</span> complete! 1 of 0 LGTMs obtained (waiting on @spilchen)



<!-- Sent from Reviewable.io -->

spilchen commented:
TFTR!

bors r+
